src/examples/meson.build

```diff
@@ -35,6 +35,7 @@ source_file = [
     'Masking.cpp',
     'InvMasking.cpp',
     'AnimateMasking.cpp',
+    'LoadPng.cpp',
```

Comment:
Currently there is a problem with the order of file list...., but by default we should stick to alphabetical ascending order.
Please put `LoadPng.cpp` in the correct order